<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(220, 224, 230);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="user_input">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
background-color:rgb(220, 224, 230);
border-radius: 5px;
border: 1px solid black;

}</string>
    </property>
    <property name="placeholderText">
     <string>    Enter your user or email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pass_input">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>360</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
background-color:rgb(220, 224, 230);
border-radius: 5px;
border: 1px solid black;

}</string>
    </property>
    <property name="placeholderText">
     <string>    Enter your password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sign_in">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>413</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(86, 126, 187);
border-radius: 5px;
border: 1px solid black;
color: white;
}

QPushButton:hover {
background-color: rgb(43, 76, 126);

}</string>
    </property>
    <property name="text">
     <string>Sign In</string>
    </property>
   </widget>
   <widget class="QLabel" name="user_label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;</string>
    </property>
    <property name="text">
     <string>User/Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="pass_label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>330</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="icon">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>150</y>
      <width>61</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/new/prefix1/icons/account_circle_FILL0_wght400_GRAD0_opsz48.svg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="invalid_text">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>470</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
